### PR TITLE
feat(dev): add gulp tasks

### PR DESCRIPTION
**Pull request by jrappen**
See **Files changed** tab for details.

---

@QingWei-Li Could you fix the previous commit messages so `gulp changelog` works as expected? I'm not sure how to do that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,46 +2,66 @@
...
+  },
   "homepage": "https://github.com/QingWei-Li/docsify-cli#readme",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/QingWei-Li/docsify-cli.git"
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 4"
+    "node": ">= 6",
```
Comment (QingWei-Li) [inline package.json]: Actually it is compatible with Node 4 and npm 3.

File: tools/locales/de.json [hunk 2]
```diff
@@ -1,7 +1,13 @@
 {
-    "completion": "Drucke shell Vervollständigungsskript",
-    "epilog": "Dokumentation",
```
Comment (QingWei-Li) [inline tools/locales/de.json]: Are you sure you want to remove it? I noticed that it exists in `en.json`

Comment (jrappen) [inline tools/locales/de.json]: I can add a translation, still looking for a fix for the problem described by @kamilchm in QingWei-Li/docsify-cli@02dec13

**Comment by QingWei-Li** [comment]
@jrappen 
Thank you for the PR. Is it done?

**Comment by jrappen** [comment]
@QingWei-Li As far as I got, yes.

Maybe you could review the chinese translations and then fix previous commit messages so they'll work with `gulp changelog` if possible?

**Comment by QingWei-Li** [comment]
I see. let me do it.

**Pull request merged**